scripts/test/lint.py

```diff
@@ -253,12 +253,26 @@ def check(self, path):
         return True
 
 
+class PackageIdNotMatchingFolderOrNuspecName(Lint):
+    name = "package ID doesn't match package folder name or nuspec file name"
+    recommendation = "make sure the package ID is the same as the package folder name and the nuspec file name"
+
+    def check(self, path):
+        dom = minidom.parse(str(path))
+        pid = dom.getElementsByTagName("id")[0].firstChild.nodeValue
```

Comment:
What about using a more readable name? we use in this file `pkg_id` for the same. Also `nodeValue` is an alias for `data`, I would use `data` to keep it consistent with the rest of the code.
```suggestion
        pkg_id = dom.getElementsByTagName("id")[0].firstChild.data
```